governance.md

```diff
@@ -1,149 +1,285 @@
+    - committed to project's mission and culture
+    - spends majority of time on Kubernetes
+  - Expectations
+    - provide overall technical guidance and vision for the project
+    - maintain the [definition of the project](https://kubernetes.io/docs/whatisk8s/)
+    - decide project structure, such as system layers (e.g., core) and repository breakdown
+    - assist SIGs in identifying areas of overlapping technical and ownership responsibility
+    - resolve technical escalations in the cases of OWNER and SIG lead disagreements 
+  - Benefits
+    - project decision makers
+    - technically can approve virtually any PRs
+    - can [Sponsor incubator repos](incubator.md)
+    - can Sponsor MAINTAINERS
+
+## Orthogonal technical roles
+
+API REVIEWER and APPROVER are called out specifically because the API is critical to the
+identity of Kubernetes and is a horizontal area that crosses directories and SIGs.
+
+- [**API REVIEWER**]:
+  - Requirements
+    - tenure requirement TBD
+    - nomination process TBD
+    - initial members TBD
+    - have written and/or reviewed Kubernetes APIs
+    - familiar enough with design, requirements, mechanics, conventions, style,
+      scope, gotchas, etc. of the API to be in `kubernetes/pkg/api` and `kubernetes/pkg/apis`
+      OWNERS files `reviewers` lists
+      - TODO: decouple design review from the firehose of PR reviews
+  - Expectations
+    - review API changes and proposals in their functional area
+    - ensure Kubernetes has a consistent [API style](contributors/devel/api-conventions.md),
+      patterns, and philosophies
+    - guide development of new APIs
+  - Benefits
+    - TBD
+    - added to [`api-reviewers`](https://github.com/orgs/kubernetes/teams/api-reviewers) github team
+- **API APPROVER**:
+  - Requirements
+    - tenure requirement TBD
+    - nomination process TBD
+    - initial members TBD (historically bgrant0607, thockin, smarterclayton, erictune)
+    - designed and reviewed several APIs in the system
+    - familiar with the design, requirements, mechanics, conventions, style,
+      scope, gotchas, etc. of the API to be in `kubernetes/pkg/api` and `kubernetes/pkg/apis`
+      OWNERS files `approvers` lists
+      - TODO: decouple design review from the firehose of PR reviews
+    - ensure Kubernetes has a consistent [API style](contributors/devel/api-conventions.md),
+      patterns, and philosophies
+    - guide development of new APIs
+    - codify new patterns for novel problems
+  - Expectations
+    - review and approve API changes and proposals in their functional area
+  - Benefits
+    - TBD
+    - added to [`api-approvers`](https://github.com/orgs/kubernetes/teams/api-approvers) github team
 
 ## SIG roles
-- **SIG Participant**: active in one or more areas of the project; wide 
-  variety of roles are represented
-- **SIG Lead**: SIG organizer
+- **SIG PARTICIPANT**: active in one or more areas of the project; wide variety of roles are represented
```

Comment:
I agree we could break these out to multiple docs.  It would be easier to digest.  What if we (after initial merge?) made a dir for governance, and separate docs for each topic?